errors/errors.go

```diff
@@ -357,7 +359,7 @@ func NormalizePanic(p interface{}) error {
 
 // Redact will replace all panic errors with a generic message
 func Redact(err error) error {
-	if HasErrorCode(err, ErrPanic.code) {
+	if hasErrorCode(err, ErrPanic.code) {
```

Comment:
Why is this different than `Is(ErrPanic, err)` ?

`ErrPanic` is not `ErrInternal` but a unique code and is the only one to be redacted, so `Is` should work here, remove legacy code paths